scripts/llnl_scripts/llnl_lc_build_tools.py

```diff
@@ -256,7 +257,12 @@ def uberenv_create_mirror(prefix, project_file, mirror_path):
     cmd  = "python scripts/uberenv/uberenv.py --create-mirror"
     cmd += " --prefix=\"{0}\" --mirror=\"{1}\"".format(prefix, mirror_path)
     cmd += " --project-json=\"{0}\" ".format(project_file)
-    return sexe(cmd,echo=True,error_prefix="WARNING:")
+    res = sexe(cmd, echo=True, error_prefix="WARNING:")
+    print "[~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~]"
+    print "[The above behavior of 'spack --create-mirror' is normal to throw many warnings]"
```

Comment:
Suggestion:
This sentence is a bit awkward. Consider rephrasing to something like:
> Note:  It is expected for 'spack --create-mirror' to throw warnings.

Also, should we print this message before and after ``spack --create-mirror`` ? (Or wrap the warnings in some way?)